Update: Journey -> ladder (#3726)
dfinity/portal: docs/tutorials/developer-journey/level-0/03-dev-env.mdx

## Overview

> Before you can begin our developer ladder, you need to set up our developer environment. A developer environment consists of tools and packages that are required to develop code projects. Usually, developer environments are stored and hosted on your local computer, but there are some situations where a virtual, web-based development environment exists.

Author: ```suggestion
Before you can begin the developer ladder, you need to set up a developer environment. A developer environment consists of tools and packages that are required to develop code projects. Usually, developer environments are stored and hosted on your local computer, but there are some situations where a virtual, web-based development environment exists.
```

#### Why does this matter?

> Many of the DFINITY public repositories are hosted on Github, such as our `examples` repository. You will be using code from this repository later in the developer ladder, so it is important to install `git` to assure that you can download the sample code pieces and follow along with the later tutorials.

Author: ```suggestion
Many of the DFINITY public repositories are hosted on Github, such as the `examples` repository used throughout this tutorial series. It is important to install `git` to assure that you can download the sample code and follow along.
```

#### Why does this matter?

> You'll use this working directory to contain the projects that you build throughout your developer ladder. This will help keep things organized in our local file structure.

Author: ```suggestion
You'll use this working directory to contain the projects that you build throughout this series. This will help keep things organized in your local file structure.
```
